README.md

```diff
@@ -36,26 +36,44 @@ Features:
 * Supports ensembles of multiple data streams, and supports common methods for temporal alignment of streams.
 
 ## Installation
-To install the latest stable release of **neuroconv** though PyPI, type:
+We always recommend installing and running Python packages in a clean environment. One way to do this is via [conda environments](https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#activating-an-environment):
+
+```shell
+conda create --name <give the environment a name> --python <choose a version of Python to use>
+conda activate <environment name>
+```
+
+To install the latest stable release of **neuroconv** though PyPI, run:
+
 ```shell
 pip install neuroconv
 ```
 
-For more flexibility we recommend installing the latest version directly from GitHub. The following commands create an environment with all the required dependencies and the latest updates:
+To install the current unreleased `main` branch (requires `git` to be installed in your environment, such was via `conda install git`), run:
```

Comment:
I think it would be better to point them to the official instalation instructions of the git project:
https://git-scm.com/downloads

Reply:
IDK, that tends to do different things than you would expect depending on the platform. Suggested added both just in case

On windows for instance, it's a standalone interpreter with it's own console (doesn't play well with standard command prompt)

You can technically add it to the system registry but behaves best with conda when installed as a separate package

Reply:
Ah I see. Ok, I did not see this before writing my comment below. Ok, so in your experience installing conda with git is the easier for windows users and hence you are making that suggestion. 

The current two-fold solution seems fine but we could also explicitly mention that we suggest git on windows. As you feel is better.